packages/babel-runtime-corejs2/helpers/esm/iterableToArray.js

```diff
@@ -1,5 +1,6 @@
 import _Array$from from "../../core-js/array/from";
 import _isIterable from "../../core-js/is-iterable";
+import _Array$isArray from "../../core-js/array/is-array";
 export default function _iterableToArray(iter) {
-  if (_isIterable(Object(iter)) || Object.prototype.toString.call(iter) === "[object Arguments]") return _Array$from(iter);
+  if (_Array$isArray(iter) || typeof iter === 'string' || typeof global.Symbol === 'function' && _isIterable(Object(iter)) || iter && 'length' in iter || typeof global.Map !== 'undefined' && iter instanceof global.Map || typeof global.Set !== 'undefined' && iter instanceof global.Set || Object.prototype.toString.call(iter) === "[object Arguments]") return _Array$from(iter);
```

Comment:
We don't need to modify these helpers. core-js already includes it's own Symbol check:

https://github.com/zloirock/core-js/blob/master/packages/core-js/internals/is-iterable.js#L7
https://github.com/zloirock/core-js/blob/master/packages/core-js/internals/well-known-symbol.js#L7